### PR TITLE
Fix Cloud Office Control Panel links

### PR DESCRIPTION
**Pull request by DonNateR**
- Fixes issue #1731 

- Change table to bulleted list

- Remove images

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-fd9e90914a/how-to/)

File: content/rackspace-email/cloud-office-control-panel.md [hunk 1]
```diff
@@ -16,20 +16,22 @@ The [Cloud Office Control Panel](https://cp.rackspace.com/Login.aspx?ReturnUrl=%
...
-| <img alt="System Status" height="65" src="{% asset_path rackspace-email/cloud-office-control-panel/systemstatus.png %}" width="85" /> | **System status** - Keep up-to-date with regular status announcements on the Control Panel home page. Update, maintenance, and outage reports are posted promptly so you have all the details.|
-| <img alt="API" height="65" src="{% asset_path rackspace-email/cloud-office-control-panel/icon_api_1.png %}" width="85" /> | **API integration** - Integrate our services into your workflow with our REST API. You'll have programmatic access to manage administrative users, Rackspace Email and Microsoft Exchange mailboxes, spam settings, and single-sign-on capabilities. |
+For more information on managing both Rackspace Email and Microsoft Exchange through the Cloud Office Control Panel see the [Cloud Office Control Panel overview](/how-to/cloud-office-control-panel-overview/).
+
+The following lists Cloud Office Control Panel feature:
```
Comment (nimbinatus) [inline content/rackspace-email/cloud-office-control-panel.md]: Should this be "The following list details the Cloud Office Control Panel features:"?

File: content/rackspace-email/cloud-office-control-panel.md [hunk 2]
```diff
@@ -16,20 +16,22 @@ The [Cloud Office Control Panel](https://cp.rackspace.com/Login.aspx?ReturnUrl=%
...
-| <img alt="Information Management" height="65" src="{% asset_path rackspace-email/cloud-office-control-panel/informationmanagement.png %}" width="85" /> | **Information management** - View and export account invoices, domain details, and email statistics at any time. |
-| <img alt="System Status" height="65" src="{% asset_path rackspace-email/cloud-office-control-panel/systemstatus.png %}" width="85" /> | **System status** - Keep up-to-date with regular status announcements on the Control Panel home page. Update, maintenance, and outage reports are posted promptly so you have all the details.|
-| <img alt="API" height="65" src="{% asset_path rackspace-email/cloud-office-control-panel/icon_api_1.png %}" width="85" /> | **API integration** - Integrate our services into your workflow with our REST API. You'll have programmatic access to manage administrative users, Rackspace Email and Microsoft Exchange mailboxes, spam settings, and single-sign-on capabilities. |
+For more information on managing both Rackspace Email and Microsoft Exchange through the Cloud Office Control Panel see the [Cloud Office Control Panel overview](/how-to/cloud-office-control-panel-overview/).
```
Comment (nimbinatus) [inline content/rackspace-email/cloud-office-control-panel.md]: Comma needed:
> ... Control Panel, see the...

File: content/rackspace-email/cloud-office-control-panel.md [hunk 3]
```diff
@@ -16,20 +16,22 @@ The [Cloud Office Control Panel](https://cp.rackspace.com/Login.aspx?ReturnUrl=%
...
+
+The following lists Cloud Office Control Panel feature:
+
+- **Fanatical Support**<sup>&reg;</sup>: Administrators can access live chat and ticket support 24x7x365. Easily access the support site for step-by-step guides.
```
Comment (nimbinatus) [inline content/rackspace-email/cloud-office-control-panel.md]: For `Fanatical Support`, we should be using bold italic: `***Fanatical Support***`

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a132fdff42/how-to/). :bow:

**Comment by nimbinatus** [comment]
LGTM, +1

**Pull request merged**